src/super_gradients/training/datasets/dataset_interfaces/dataset_interface.py

```diff
@@ -815,27 +818,38 @@ def __init__(self, dataset_params={}):
         local_rank = get_local_rank()
         with wait_for_the_master(local_rank):
             self.trainset = COCODetectionDataset(data_dir=self.dataset_params.data_dir,
-                                                 name=self.dataset_params.train_subdir,
+                                                 subdir=self.dataset_params.train_subdir,
                                                  json_file=self.dataset_params.train_json_file,
-                                                 img_size=train_input_dim,
+                                                 input_dim=train_input_dim,
                                                  cache=self.dataset_params.cache_train_images,
-                                                 cache_dir_path=self.dataset_params.cache_dir_path,
+                                                 cache_dir=self.dataset_params.cache_dir,
                                                  transforms=train_transforms,
+                                                 tight_box_rotation=train_tight_box_rotation,
+                                                 class_inclusion_list=self.dataset_params.class_inclusion_list,
+                                                 max_num_samples=train_max_num_samples,
                                                  with_crowd=False)
 
         val_input_dim = (self.dataset_params.val_image_size, self.dataset_params.val_image_size)
+        val_tight_box_rotation = get_param(self.dataset_params, "val_tight_box_rotation", False)
```

Comment:
Same comment regarding the defaults here...
Also- does tight_box_prediction have a meaning when dealing with validation set ?

Reply:
val_tight_box_rotation removed in f1dc9d6f317f30c415d7c26cb912c4ebc2287307 because not relevant for validations tep